test/functional/managedcluster_test.go

```diff
@@ -409,34 +459,7 @@ var _ = Describe("Managedcluster", func() {
 
 				By("checking ManagedCluster Creation")
 				checkManagedClusterCreation(clientHubDynamic, clientHub, myTestNameSpace, myTestNameSpace, gvrManagedcluster, gvrServiceaccount, gvrSecret)
-				managedCluster = setManagedClusterConditionAvailable(myTestNameSpace, true)
-				Eventually(func() error {
-					klog.V(1).Info("Waiting managedCluster auto import condition")
```

Comment:
we don't need this test case any more?

Reply:
No, this now embedded in setManagedClusterConditionAvailable as we wait for that condition no need to test it again